pages/index.tsx

```diff
@@ -1,27 +1,29 @@
 import { useState } from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import Head from 'next/head';
 
 import NFTView from '@/view/NFTView';
 
-const TAB_INFORMS = [
+const TAB_IN_FORMS = [
```

Comment:
복수형이 FORM 보다는 TAB쪽에 들어가는게 더 적절할 것 같아요~

```suggestion
const TABS_IN_FORM = [
```